### PR TITLE
Add L0 request rate test

### PR DESCRIPTION
**Pull request by nv-braf**

### CODE REVIEW

File: qa/L0_profile_request_rate/test_config_generator.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            '--profile-models',
+            type=str,
+            required=True,
+            help='Comma seperated list of models to be profiled')
+        parser.add_argument('-b',
+                            '--batch-sizes',
+                            type=str,
+                            required=True,
+                            help='List of batch sizes')
+        parser.add_argument('-r',
```
[the remark this answers is not in the document]
Comment (nv-braf) [reply qa/L0_profile_request_rate/test_config_generator.py:54]: This is just the YAML for the L0 test.

[the remark this answers is not in the document]
Comment (tgerdesnv) [reply qa/L0_profile_request_rate/test_config_generator.py:54]: Yeah, I'm still wondering what the `-r` is for?  Does it do anything? Is it used anywhere?

[the remark this answers is not in the document]
Comment (nv-braf) [reply qa/L0_profile_request_rate/test_config_generator.py:54]: It's used in `test.sh` to specify the request rate to the `test_config_generator` (in the standard profile test it's `-c` for concurrency)

[the remark this answers is not in the document]
Comment (tgerdesnv) [reply qa/L0_profile_request_rate/test_config_generator.py:54]: Got it. Thanks.

**Review by tgerdesnv — APPROVED** [review]

**Pull request merged**